types/test.tsx

```diff
@@ -149,19 +149,25 @@ const actions = {
 };
 
 // @ts-expect-error
-const TypeStore0 = createStore<State, Actions>({ count: 0 });
+const BasStore = createStore<State, Actions>({ count: 0 });
 
-const TypeStore1 = createStore<State, Actions>({
+createStore<State, Actions>({
   // @ts-expect-error
   initialState: { bla: 0 },
   actions,
 });
 
 // @ts-expect-error
-const TypeStore2 = createStore<State, Actions>({ initialState: { count: 0 } });
+createStore<State, Actions>({ initialState: { count: 0 } });
 
 // @ts-expect-error
-const TypeStore3 = createStore<string, Actions>({ initialState: '', actions });
+createStore<string, Actions>({ initialState: '', actions });
+
+// @ts-expect-error
+createStore({ containedBy: createContainer(createStore(BasStore)) });
```

Comment:
Not sure I get this one?

Reply:
insufficient arguments

Reply:
why passing a store to createStore though?

Reply:
This test is to ensure `containedBy` container is not an override one. Made it more clear